![](_page_0_Picture_1.jpeg)

Contents lists available at [ScienceDirect](www.sciencedirect.com/science/journal/02235234)

# European Journal of Medicinal Chemistry

journal homepage: <http://www.elsevier.com/locate/ejmech>

Research paper

# Synthesis, biological evaluation, QSAR and molecular dynamics simulation studies of potential fibroblast growth factor receptor 1 inhibitors for the treatment of gastric cancer

![](_page_0_Picture_7.jpeg)

Shilong Ying a, <sup>1</sup> , Xiaojing Du a, b, <sup>1</sup> , Weitao Fu a, <sup>1</sup> , Di Yun <sup>a</sup> , Liping Chen <sup>a</sup> , Yuepiao Cai <sup>a</sup> , Qing Xu <sup>c</sup> , Jianzhang Wu a, \* , Wulan Li a, d, \*\*, Guang Liang <sup>a</sup>

<sup>a</sup> Chemical Biology Research Center, School of Pharmaceutical Sciences, Wenzhou Medical Universtiy, Wenzhou, Zhejiang 325035, China

<sup>b</sup> Department of Digestive Diseases, The First Affiliated Hospital of Wenzhou Medical University, Wenzhou, Zhejiang 323000, China

<sup>c</sup> College of Chemistry and Materials Engineering, Wenzhou University, Wenzhou, Zhejiang 325035, China

<sup>d</sup> College of Information Science and Computer Engineering, Wenzhou Medical University, Wenzhou, Zhejiang 325035, China

## article info

Article history: Received 27 August 2016 Received in revised form 12 October 2016 Accepted 31 October 2016 Available online 1 November 2016

Keywords: Design Fibroblast growth factor receptor 1 Gastric cancer Quantitative structure-activity relationship

## abstract

Accumulating evidence suggests that fibroblast growth factor receptor 1 (FGFR1) is an attractive target in gastric cancer therapy. Based on our previous discovery of two non-ATP competitive FGFR1 inhibitors, A114 and A117, we designed and screened a series of compounds with the framework of bisaryl-1,4-dien-3-one. Among them, D12 and D15 exhibited the most potent FGFR1 inhibitory activity, which was ATPindependent. Furthermore, a quantitative structure-activity relationship analysis of 41 analogs demonstrated that the specific structural substitutions alter their bioactivities. Molecular docking and dynamics simulation analysis indicated the hydrophobic interaction at the FGFR1-D12/D15 interaction was dominant. Evaluation for anti-gastric cancer efficacy of D12 and D15 indicated effective inhibition of cell proliferation, apoptosis induction and cell cycle arrest. Thus, these two FGFR1 inhibitors have therapeutic potential in the treatment of gastric cancer, and this study provides will contribute to the rational design of novel non-ATP competitive FGFR1 inhibitors.

© 2016 Elsevier Masson SAS. All rights reserved.

#### 1. Introduction

Gastric cancer (GC) is the fifth most common malignancy and the third leading cause of cancer-related death worldwide [\[1,2\].](#page-14-0) Although surgery procedures and chemotherapy have improved greatly in recent years, the clinical outcomes with treatment of gastric cancer remain poor due to absence of early detection of the cancer [\[3\].](#page-14-0) For advanced gastric cancer patients, a combination of surgery and traditional chemotherapy is still the mainstream remedy [\[4,5\].](#page-14-0) However, the severe and irreversible side-effects of chemotherapy limit their therapeutic use [\[6,7\]](#page-14-0). Thus, development of newly targeted therapy for gastric cancer with minimal sideeffects is highly desirable. At present, the only small molecule

<sup>1</sup> These authors contribute to this work equally.

<http://dx.doi.org/10.1016/j.ejmech.2016.10.066> 0223-5234/© 2016 Elsevier Masson SAS. All rights reserved. inhibitor for GC therapy available on the market is Apatnib, an inhibitor of vascular endothelial growth factor receptor (VEGFR) which was approved by the China Food and Drug Administration (CFDA) in 2014 [\[8,9\]](#page-14-0). In recent years, an increasing number of signaling molecules are considered as novel targets against cancer [\[10\]](#page-14-0). Of these, Fibroblast growth factor receptor 1 (FGFR1) is considered to be an important potential target for treating GC [\[3,11,12\].](#page-14-0)

FGFR1, belonging to the superfamily of receptor tyrosine kinases (RTK), has been intensively studied as a drug target for cancer [\[13,14\].](#page-14-0) The binding of fibroblast growth factor (FGF) and FGFR triggers the dimerization of the extracellular receptor domains, trans-phosphorylation of the intracellular kinase domain, and the subsequent activation of downstream signaling pathways, the mitogen activated protein kinase (MAPK) and phosphoinositide-3 kinase (PI3K)/Akt [\[15,16\]](#page-14-0). Increased activity of FGFR1 is closely linked with malignant tumors, such as lung cancer [\[17\],](#page-14-0) gastric adenocarcinoma [\[12\],](#page-14-0) renal cell carcinoma [\[18\]](#page-14-0), and breast cancer [\[19\]](#page-14-0). Aberrant FGFR1 activity as a result of mutations, gene amplifications, or chromosomal translocations are reported as a key

c1ccccc1

c1ccccc1

c1ccccc1

<sup>\*</sup> Corresponding author.

<sup>\*\*</sup> Corresponding author. Chemical Biology Research Center, School of Pharmaceutical Sciences, Wenzhou Medical Universtiy, Wenzhou, Zhejiang 325035, China.

E-mail addresses: [wjzwzmu@163.com](mailto:wjzwzmu@163.com) (J. Wu), [lwlwzmu@163.com](mailto:lwlwzmu@163.com) (W. Li).

driver of tumor progression by promoting proliferation, angiogenesis, survival, migration and invasion of cancer cells [\[20,21\].](#page-14-0) Recently, Oki et al. found that FGFR1 expression is upregulated in GC tissue specimens [\[22\],](#page-14-0) as are several GC cell lines (MKN-74, MKN-45, BGC-823, MGC-803, and SGC-7901) [\[23\].](#page-14-0)

At present, several small molecule inhibitors of FGFR1 are being evaluated in pre-clinical or clinical trials as anti-gastric cancer agents [\[13,24,25\]](#page-14-0). In one study, NVP-BGJ398, a pan-FGFR inhibitor showed strong inhibition of proliferation of KKLS cells which have high FGFR1 expression, and built with low inhibition on TMK-1 cells expressing low levels of FGFR1 [\[25\].](#page-14-0) Another potent FGFR1 inhibitor, AZD4547, has been completed a phase 2 clinical trial assessing its efficacy and safety in advanced gastric or gastrooesophageal junction cancer patients (NCT01795768). However, the development of current FGFR inhibitors as anti-cancer drugs remains limited. Most of these inhibitors belong to ATPcompetitive inhibitors, which bind to and inhibit the relatively conserved ATP binding domain of RTKs. Such inhibition of FGFR often results in significant side-effects and toxicity in clinical and pre-clinical studies [\[26](#page-14-0)e[28\].](#page-14-0) Thus, the development of non-ATPcompetitive FGFR inhibitors can be a more selective approach to overcome the above difficulties.

Previously, we identified two bisaryl-1,4-dien-3-one-containing non-ATP competitive FGFR1 inhibitors, A114 and A117, and characterized their therapeutic potency in non-small cell lung cancer in vitro and in vivo studies [\[29\]](#page-14-0). To identify FGFR1 inhibitors with greater efficacy for gastric cancer, we analyze the structureactivity relationship of analogs of bisaryl-1,4-dien-3-one-containing FGFR1 inhibitors. We developed four series of analogs by a rational modified strategy based on the inhibitory activity screening results (Fig. 1). We also established a QSAR model to identify the pharmacophore. Firstly, we altered the cyclopentanone ring in the structure of A114 and A117 into various ketones (the A series compounds) to investigate whether the middle-linking chain was essential. Moreover, further detailed modifications were made on optimization of one of the phenyl substituents to develop the B series compounds. The latter was the unilateral 4-nitrogen atomcontaining moiety, which significantly enhanced the activity of the compound. To confirm the effect of this interesting active fragment, the C series compounds, characterized by a common para-position nitrogen atom-containing heterocyclic group, was produced. By removing the propionyl or isobutyryl fragment to produce an exposed hydroxyl, the D series compound was designed, which was expected to effectively elevate the activity by forming hydrogen bond donors and acceptor.

In this study, we successfully screened and found two active FGFR1 inhibitors, D12 and D15, by kinase inhibition assay and confirmed their non-ATP competitive inhibitory characteristic.

![](_page_1_Figure_7.jpeg)

Fig. 1. Chemical structure of A114, A117 and design strategy of four series derivatives.

c1ccccc1

c1ccccc1

c1ccccc1

Biological assessment indicated effective suppression of GC proliferation and induction of apoptosis. This finding constructs an appropriate QSAR model of non-ATP competitive FGFR1 inhibitors for the first time, and provides two potential targeted therapeutic agents against the GC.

#### 2. Results and discussion

#### 2.1. Chemistry

The synthetic routes of the 41 derivatives of A114 and A117 were shown as [Scheme 1.](#page-3-0) The synthesis of A1-A7 started from corresponding ketones 1 (1a: tetrahydro-4H-pyran-4-one, 1b: cyclohexanone, 1c: acetone, 1d: 4-Piperidone hydrochlorides hydrate). The intermediates 3a-3g were obtained through a step of the classic Claisen-Schmidt condensation reaction, which applied HCl gas as catalyst, and then reacted with propionyl or isobutyryl chloride under Et3N catalyzing to prepare the compounds A1-A7. The procedures to prepare B1-B14, C1-C5 and D1-D13 were similar, in which cyclopentanone 1e or cyclohexanone 1b was activated with morpholine to generate the enamines 5e or 5b. The ethanol solution of 5e or 5b with vanillin was reacted under 78 C, adjusted the pH to acidic conditions to obtain the key intermediates 7e or 7b. Next, the solutions were condensed with different substitutional aromatic aldehyde for preparing the desired D1-D13, 8e1-8e14 and 8b1-8b5. Further, acylation of 8e1-8e14 and 8b1-8b5 with propionyl or isobutyruyl in the presence of Et3N produced the B1eB14 or C1eC5. The general process for synthesizing D14-D15 was by a three-step reaction. Protection of Vanillin with 3,4-dihydro-2Hpyran (THP) yielded the corresponding benzaldehyde 10. Through a unilateral aldol condensation between 10 with acetone in a catalyst of 20% NaOH achieved the conversion to unilateral unsaturated ketone 11. Finally, D14-D15 was obtained by Claisen-Schmidt condensation between 11 and corresponding benzaldehydes in the presence of 40% NaOH and the deprotection of THP in an acidic environment. All compounds were purified by column chromatography. The structures of A-D series compounds were shown in [Scheme 1,](#page-3-0) and confirmed by spectral analysis by high-resolution mass spectrometry (HRMS), nuclear magnetic resonance (1 H-NMR and 13C-NMR) spectroscopy.

#### 2.2. The inhibitory activity of synthetic compounds on FGFR1 kinase

The synthesized bisaryl-1,4-dien-3-one-containing compounds were initially screened for inhibition of FGFR kinase activity at a concentration of 20 mM by a Mobility Shift Assay. As shown as in the [Scheme 1,](#page-3-0) compound B14 exhibited the most potent inhibitory activity of FGFR1 kinase, and approximately half of the compounds (18/41) exhibited inhibitory ratio greater than 50%. These compounds were chosen for the further determination of half-maximal inhibitory concentration (IC50).

Based on the inhibitory potency of all compounds (at 20 mM), a primary structure-activity relationship (SAR) was analyzed. For the A series compounds, the middle aliphatic ketone linker exerted a crucial influence on the inhibitory activity of compounds, changing the middle ketone into either pyrone or piperidone, abolished the potent action of A114 and A117. Based on our previous molecular docking study [\[29\]](#page-14-0), the middle connecting ketone group of A114 and A117 was likely sandwiched in a hydrophobic cleft of FGFR1 kinase domain. An aliphatic ketone linker can greatly stabilize the inactive conformation of kinase domain, as well as provide the selective inhibition on FGFR1.

It's noteworthy that compound (B1), which adopted a cross position exchange between the propionyloxyl and methoxyl group, or a replacement of the propionyloxyl group with a methoxyl group (B2), resulted in a complete activity loss, which demonstrating that the inhibitory effect of A114 and A117 was closely related to the substituents on their benzene rings. Whether introducing electrondonating (methoxyl, ethyoxyl) or electron-withdrawing substituents (F atom) in one benzene ring of A114 and A117, either contributed little to activity, other than with the 4-nitrogen atomcontaining substituents. Significantly, results indicated that compounds bearing 4-pyrrolidyl or 4-morpholinyl (B7, B13 and B14) exhibited outstanding potency among the B series compounds, providing an attractive active fragment for the design of novel structural FGFR1 inhibitors based on the bisaryl-1,4-dien-3-one scaffold. The screening for the asymmetric derivatives (B, C, and D series) revealed that introducing the 4-nitrogen atom-containing substituents could be useful to enhance the inhibition efficacy. The C series compounds displayed a high level of inhibitory activity, with inhibitory ratio of IR % and a range of 84.3%e94.0% at a compound concentration of 20 mM. Moreover, compounds with the 4-nitrogen atom-containing group on the unilateral benzene ring or a 3-substituted indole moiety effectively suppressed FGFR1 kinase activity (D9-D15, IR %: 50.7%e87.1%, 20 mM).

## 2.3. Quantitative evaluation of structure-activity relationship (QSAR)

A QSAR model was constructed to further investigate the SAR of the derivatives of A114 and A117. Small molecules can be represented as molecular descriptors including constitutional descriptors, topological descriptors, auto-correlation descriptors, charge descriptors, molecular properties, which capture and magnifies distinct aspects of chemical structures. Scatter plot of the predicted activity versus experimental values was displayed in [Fig. 2](#page-4-0). Relative high regression coefficient was obtained from the ChemoPy descriptor calculation program and R program. It had a high adjusted squared regression coefficient (R<sup>2</sup> adj ¼ 0.8196) and described more than 83.31% of variances in the experimental activity. Equation listed in the figure showed that MoRSEC27 (weighted by atomic charge) and bcute5 (weighted by atomic Sanderson electronegativities) created the major influence in the model. The inhibitory activities of molecules were negatively regulated by MoRSEC27, and positively regulated by bcute5, while S35 (Sum of E-State of atom type: dO) showed little relevance. Consequently, the electronegativity may exert a critical influence on the FGFR1 inhibitory activity of compounds. The combined SAR and QSAR results may provide useful information for the future development of novel FGFR1 kinase inhibitors.

## 2.4. D12 and D15 inhibit FGFR1 through an ATP-independent mechanism

In our early research, we identified the non-ATP competitive inhibitory mode of A114 and A117. Thus, the effect of ATP concentration on the potency of inhibition of FGFR1 activation by D12 and D15 was evaluated by a Caliper Mobility Shift Assay to determine whether the inhibitory mechanism was the same as with the parent compound. With increasing ATP concentration, the rate of phosphorylation of FGFR1 substrate gradually increased, reaching a plateau with ATP at 1048 mM. In the presence of D12 up to 20 mM, most of phosphorylation was abolished, and further increase in ATP concentration, up to 4192 mM, had no effect on the inhibitory potency of D12 [\(Fig. 3\)](#page-4-0), The finding supported a likely non-ATP competitive inhibition mechanism. We previously reported that the inhibition effect of PD173074, an ATP-competitive FGFR1 inhibitor, markedly reduced with increasing ATP concentration, indicating competition between ATP with the inhibitor [\[29\]](#page-14-0). Similarly, we evaluated D15 using the ATP-competitive assay and found

c1ccccc1

c1ccccc1

c1ccccc1

<span id="page-3-0"></span>![](_page_3_Figure_1.jpeg)

**Reagents and conditions:** (a) HCl gas, EtOH, 40%–50%; (b) THF, propionyl, or isobutyryl chloride, Et3N, rt, 8%–31%; (c) p-TSA, cyclohexane, reflux, 50%; (d) EtOH, 90 °C, 30%; (e) 10% HCl solution, rt, 50%; (f1, f2) differently aldehydes, HCl gas, EtOH, 50–70 °C, 10%–70%; (g1, g2) THF, propionyl or isobutyryl chloride, Et3N, rt, 5%–54%; (h) PPTS, CH2Cl2, rt, 50%; (i) 20% NaOH solution, EtOH, rt, 50%; (j) substituted aldehydes, NaOH/dioxane suspension, 10%–70%; (k) 10% HCl solution, rt, 50%;

**Abbreviation:** HCl, hydrochloric acid; EtOH, ethanol; THF, tetrahydrofuran; Et3N, triethylamine; rt, room temperature; p-TSA,para-toluenesulfonic acid; IR%, inhibitory ratio at a concentration of 20μM.

Scheme 1. General synthetic route for the target compounds.

c1ccccc1

c1ccccc1

c1ccccc1

<span id="page-4-0"></span>![](_page_4_Figure_1.jpeg)

Fig. 2. Plots of predicted activity against the corresponding experimental activity on FGFR1 inhibition.

Abbreviation: MoRSEC27, 3D-MoRSE - signal 27/weighted by atomic charge; S35, Sum of E-State of atom type: dO; bcute5, Highest eigenvaluen.5 of Burden matrix/weighted by atomic Sanderson electronegativities; N, the number of compounds taken into account in the regression; R2 , the multiple correlation coefficient; Radj <sup>2</sup> , adjusted multiple correlation coefficient; s, residual standard error; and the F value is related to the F-statistic analysis (Fischer test). The numbers in parentheses mean the standard deviation of the coefficients. p means the significance of the variables in the model.

ATP-independent mechanism of inhibition. Based on these results, we conclude that the mechanism of action of D12 and D15 was in an ATP-independent, and consistent with their leads A114 and A117.

## 2.5. Molecular docking and molecular dynamics simulation

We investigated interactions between the compounds and the kinase domain of FGFR1 by molecular docking analysis of the active compound D12/D15. We docked D12/D15 into the inactive conformation of FGFR1 kinase using our well-established molecular docking method which is based on the FGFR1-ARQ069 complex structure for reference (PDB Code: 3RHX) [\[28\].](#page-14-0) From an overall determination of root mean square deviations (RMSD) and estimated free energy, three poses of docking results were chosen as initial structures for molecular dynamics (MD) simulation. For D12, 93% poses with RMSD less than 2 Å, and the top ranking pose was chosen. For D15, there were two types of binding modes, one occupied 23% and others were 72%. Hence, two poses of D15 were chosen as initial structures for MD simulation.

The RMSD of the receptor in FGFR1/D12 and FGFR1/D15 complexes was converged after 10e20 ns in 50 ns MD simulations, and the RMSD of backbone atoms was z 1e2 Å and z 4e5 Å, respectively [\(Fig. 4](#page-5-0)AeB). Binding energies for D12/D15 were calculated by MM-GBSA using 500 snapshots from 40 to 50 ns stable trajectories. Two poses of D15 were carried out for MD simulation. One showed 43.8067 ± 2.6494 kcal/mol binding energy, another was 26.9956 <sup>±</sup> 3.6117 kcal/mol. Therefore, D12 and the significantly lower binding energy (high binding affinity) pose of D15 was chosen for further energy decomposition. In [Fig. 4](#page-5-0)C and D, the results showed that D12 and D15 shared seven identical key residues: PHE498, LEU630, ALA640, MET535, ASP641, VAL492 and ILE545. Interestingly, except ASP641, all these residues were hydrophobic amino acids, indicating that D12 and D15 likely interact primarily with non-polar residues in FGFR1 to exert their inhibitory activity. Kinases can exist as active and/or inactive states in cells. The hydrophobic residues in kinases interact to form "hydrophobic clusters" that stabilize the inactive conformation and interfere with ATP binding [\[26,33\]](#page-14-0). Targeting the auto-inhibited conformation (inactive conformation) of the kinase by the inhibitors represents an attractive approach due to the possibility of discovering more selective inhibitor [\[33\].](#page-14-0) For example, ARQ069, previously identified to be a non-ATP-competitive inhibitor of FGFR1/2, binds and inhibits the auto-inhibited conformation of FGFR1, and displays favorable kinase selectivity [\[28\]](#page-14-0).

[Fig. 5A](#page-6-0) shows the whole molecule of D12 was sandwiched in a hydrophobic cleft between the residues PHE489, ALA640, LEU630, VAL492, VAL561, IEL545, and PHE642. The phenyl ring of indolyl in D12 created p-p stacking with the benzene side chain of PHE489 to stabilize the binding conformation, and another phenyl ring occupied the main hydrophobic pocket by making hydrophobic contacts with residues ILE545, MET535, PHE642, and ALA640. The results indicated that D12 formed one hydrogen bond with the side chain of PHE642. We compared modeling of the binding of D12 into FGFR1 with the FGFR1-ARQ069 co-crystal complex structure. D12 was observed to possess highly similar interactions with ARQ069, while ARQ069 only made two hydrogen bond interactions with the kinase domain, the major binding feature of ARQ069 was hydrophobic interactions, other than hydrogen bond interactions. The simulation results of D15 was almost identical to D12 ([Fig. 5B](#page-6-0)). The left phenyl ring of D15 formed p-p stacking with PHE489 as well. In addition, the right phenyl ring was wrapped in a hydrophobic pocket comprised of VAL561, ILE545, ALA640, and MET535. An extraordinary phenomenon was that no hydrogen bond existed between D15 with the kinase domain, suggesting that D15

![](_page_4_Figure_10.jpeg)

Fig. 3. D12 and D15 inhibit FGFR1 through a mechanism that is independent of the concentration of ATP. Selective ATP-competitive kinase assay of compounds D12 and D15 with FGFR1 through Caliper Mobility Shift Assay. The conversion data were fitted with Graphpad for global fitting.

c1ccccc1

c1ccccc1

c1ccccc1

c1ccccc1

<span id="page-5-0"></span>![](_page_5_Figure_1.jpeg)

Fig. 4. Backbone RMSDs are shown as a function of time for FGFR1/D12 and FGFR1/D15 complexes structures at 50 ns? A) Time evolution of the RMSD of FGFR1 and D12 are shown with green lines and blue lines, respectively. (B) Time evolution of the RMSD of FGFR1 and D15 are shown with green lines and blue lines, respectively. (C, D) Per-residue of top 10 contribution to the binding effective energy of D12/D15. Per-residue contributions were calculated by the MM-GBSA decomposition method. (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this article.)

c1ccccc1

c1ccccc1

<span id="page-6-0"></span>![](_page_6_Figure_1.jpeg)

Fig. 5. Molecular dynamics analysis of D12/D15 to the activity cavity of FGFR1. (A) Last snapshot of FGFR1/D12 in 50ns MD simulations. (B) Last snapshot of FGFR1/D15 in 50ns MD simulations.

interacted with FGFR1 kinase by only forming stable hydrophobic interactions. In summary, a critical hydrophobic interaction between D12 (and D15) with FGFR1 kinase domain was predicted by a rational method combined with molecular docking and molecular dynamics simulation. These methods clarified the interaction mechanisms of D12 and D15 with FGFR1. The further verification of co-crystallization of FGFR1-D12 and FGFR1-D15 is in progress.

## 2.6. D12 and D15 suppress the growth of GC cell lines

Aberrant activation of FGFR1 is correlated with tumor growth and development in a number of different cancers, including gastric cancer. The most potent compounds for evaluation of GC growth were selected based on their IC50 values. Most compounds showed favorable inhibitory activities using single concentration screening, with only four compounds showing relative low kinase IC50 (<10 mM) [\(Fig. 6\)](#page-7-0). The effect of the four FGFR1 kinase inhibitors (C5, D12, D14, and D15) on three GC cell lines was determined using the MTT assay. The GC cell lines were treated with each of the compounds, or with A114, A117, AZD4547 and Nordihydroguaiaretic acid (NDGA), a non-ATP competitive FGFR1 inhibitor, as the positive controls for 72 h. Among all tested compounds, D12 and D15 exhibited the greatest inhibitory activity of the MGC803 and BGC823 cell lines, with IC50 values of D12 and D15 lower than 10 mM in growth inhibition, and the anti-proliferative efficacies greater than their leads [\(Fig. 6](#page-7-0)).

## 2.7. D12 and D15 inhibit the phosphorylation of FGFR1 and the downstream signaling

FGF binding to FGFR1 induces the phosphorylation of FGFR1 and activates downstream pathways, including extracellular signalregulated kinase (ERK). ERK is a member of mitogen activated protein kinase (MAPK) and crucial for cellular proliferation and differentiation [\[34\]](#page-14-0). We determined the inhibitory effects of D12 and D15 on the phosphorylation of FGFR1 and ERK by Western blot

c1ccccc1

c1ccccc1

| 化合物   | SMILES       |
|-------|--------------|
| 化合物-1 | C1=CC=CC=C1O |
| 化合物-2 | C1CCC(CC1)O  |

Fig. 6. Active compounds (C5, D12, D14 and D15) effectively inhibited the FGFR1 kinase activity and the proliferation of three GC cells (SGC7901, MGC803, and BGC823). The IC50 values of the tested compounds on both enzyme and cell level were as shown. Four reported FGFR inhibitor A114, A117, AZD4547 and NDGA were used as a comparison in MTT assay. N.D. meant not determined. GC cells (3 103 cells/well; 96-well plates) was pretreated with compounds for 72 h, then the MTT assay gives the respective IC50 values of each compounds. The data were shown as a mean of 3e5 independent tests.

analysis. Fig. 7 showed that D12 and D15 significantly suppressed the FGF2-induced phosphorylation of the threonine (T653/654) of FGFR1 and ERK activation in a dose-dependent manner.

## 2.8. D12 and D15 arrest the cell cycle at G0/G1 phase

We further examined whether D12 and D15 possessed cell cycle arrest potential. The SGC7901 cells were treated with different concentrations of D12 and D15 (5 mM, 10 mM, 20 mM) for 8 h, and cell cycle population distribution was measured by flow cytometry. Results indicated that D12 and D15 both induced cell cycle G0/G1 phase arrest in a concentration-dependent manner, and dramatically reduced the cell population of the G2/M phase ([Fig. 8A](#page-8-0)). The up-regulation of cyclin D1 can be an oncogene driving the cell-cycle regulation [\[35\],](#page-14-0) which has been reported to overexpressed in gastric cancer cells [\[36\]](#page-14-0). Therefore, we determined the expression level of cyclin D1 by Western blot analysis. Results indicated that both D12 and D15 down-regulated cyclin D1 expression in a dosedependent manner ([Fig. 8](#page-8-0)B). In summary, D12 and D15 induced cell cycle arrest of gastric cancer cells, which was attributed to at least in part by reducing cyclin D expression.

## 2.9. D12 and D15 induce apoptosis of GC cell lines

We determined whether the GC cell death induced by D12 and D15 was attributed to apoptosis through the mitochondriamediated apoptotic pathway. The morphological changes of cell nuclei was determined by Hoechst staining, and the expression of apoptosis-related molecules, i.e., cleaved-PARP and anti-apoptotic factor Bcl-2, was detected by Western blot analysis. Following 12 h treatment of GC cells with compounds, Hoechst staining showed chromatin condensation (strong blue fluorescence) and nucleus fragmentation ([Fig. 9](#page-9-0)A). At increasing higher concentrations of the compounds concentration, chromatin condensation developed in a concentration-dependent manner. For Western blot analysis, SGC7901 cells were incubated with different concentrations of D12, D15 (5 mM, 10 mM, 20 mM) or 10 mM of A117. Results indicated that the expression of cleaved-PARP was increased, which was accompanied by decreased expression of Bcl-2, both in a dosedependent manner [\(Fig. 9B](#page-9-0)). Note that as low as 5 mM of D12 and D15 was sufficient to induce changes of the protein expression. Compared with A117, D12 and D15 achieved the similar effects at 10 mM [\(Fig. 9](#page-9-0)B). These results indicated that D12 and D15 were effective inducers of GC cells apoptosis.

#### 3. Conclusion

In summary, with A114 and A117 as lead compounds, four series of derivatives were designed and synthesized. Among them, D12 and D15 were screened by kinase inhibition assay, and identified as non-ATP competitive inhibitors of the kinase domain of FGFR1. QSAR analysis confirmed that the inhibition potency was highly

![](_page_7_Figure_12.jpeg)

Fig. 7. Compounds D12 and D15 suppressed bFGF-induced phosphorylation of FGFR1 and the downstream ERK1/2 in a concentration-dependent manner. SGC7901 cells were incubated with compounds D12 (5, 10, 20 mM), D15 (5, 10, 20 mM), 10 mM A117 or DMSO respectively, and then stimulated with bFGF; cell lysates was collected and the phosphorylation levels of FGFR1 and ERK1/2 were determined by the western blot assay.

c1ccccc1

c1ccccc1

c1ccccc1

c1ccccc1

<span id="page-8-0"></span>![](_page_8_Figure_1.jpeg)

Fig. 8. Compound D12 and D15 induced cell cycle arrest at the G0/G1 phase in SGC7901 cells. (A) Cell cycle analysis. Cells was treated with various concentrations of D12 and D15 for 8 h, then subjected to flow cytometric analysis for the distributions at each phase of cells. (B) Western blot analysis of cell cycle protein Cyclin D1. Data are presented as the mean ± SD of three independent experiments conducted in triplicate. \*P < 0.05; \*\*P < 0.01; \*\*\*P < 0.001.

correlated with the skeleton structure and special substituents in the synthetic compounds. Molecular docking and molecular dynamics simulations analyses demonstrated that the binding mode of D12 and D15 with the kinase domain of FGFR1 was most likely hydrophobic interactions. Moreover, biological assays of antigastric cancer efficacy of D12 and D15 indicated effective inhibition of cell proliferation, apoptosis induction and cell cycle arrest. Based on these findings, we conclude that D12 and D15 are potential agents for treatment of gastric cancer. This work provided a reference for the discovery of novel FGFR1 inhibitors.

## 4. Experimental section

#### 4.1. Chemistry

Reagents and solvents for the synthesis were commercially available and obtained from Sigma-Aldrich (St Louis, Missouri, USA) and Aladdin (Shanghai, China), which were used without further purification. Silica gel (GF254) for column chromatography (200e300 mesh) was obtained from Aladdin. Melting points were measured on a Fisher-Johns melting apparatus and were

c1ccccc1

c1ccccc1

c1ccccc1

c1ccccc1

<span id="page-9-0"></span>![](_page_9_Figure_2.jpeg)

Fig. 9. (A) Morphological changes and Hoechst staining were observed in SGC7901 cells cultured with D12, D15 (5, 10, and 20 <sup>m</sup>M) for 12 h (200). The figures were representative of more than three separate experiments (\*P < 0.05, \*\*P < 0.01, \*\*\*P < 0.001). (B) Effect of D12 and D15 on the activation of PARP and inhibition of Bcl-2 in SGC7901 cells. SGC7901 cells were incubated with D12 and D15 (5, 10 and 20 mM, respectively) for 48 h. A117 (10 mM) was used as a positive control. The level of PARP and Bcl-2 after administration was assayed by the western blot analysis. The column figure was the normalized optical density as a percentage of the relevant total protein.

uncorrected. The <sup>1</sup> H-NMR and 13C-NMR spectra data were recorded on a 600 MHz spectrometer (Bruker Corporation, Switzerland) with TMS as an internal standard. Part of the synthetic compounds had been screened for their anti-inflammatory properties, and the

structural spectral data reported in our previous papers [\[30](#page-14-0)e[32\].](#page-14-0) In this article, we only listed the spectral data of unreported compounds (B7, B14, C1eC5, and D13-D15). High-resolution mass spectra (m/z) were recorded on a micrOTOF-Q II instrument. The

c1ccccc1

c1ccccc1

purity of all compounds was detected by HPLC (column: Agilent Eclipse XDB C18 5 <sup>m</sup>m 4.6 mm 150 mm, flow: 1 mL/min, detected wavelength (DW): 420 nm or 450 nm, condition: methanol/water from 50/50 to 90/10). The retained time (TR) and purities of unreported compounds in this article were collected.

## 4.1.1. General procedure for preparation of the intermediates 3a-3d

A mixture of Vanillin (2, 10 mmol) and appropriate ketone (5 mmol, 1a: tetrahydro-4H-pyran-4-one, 1b: cyclohexanone, 1c: acetone) in ethanol (20 mL) was stirred at room temperature. HCl (gas) was bubbled into the solution to catalyze the reaction until it was completed. 4-Piperidone hydrochloride hydrate (1d) and Vanillin were dissolved in the mixture solvent of ethanol and water (10:1) different with others. The resulting mixture was cooled and poured into cold water (20 mL) to precipitate the product. The filter residue was purified by silica gel chromatography using PE/EA as eluent to afford the intermediates 3a-3d.

## 4.1.2. General procedure for synthesis of compounds A1-A7

A solution of propionyl or isobutyryl chloride (3 mmol) in THF (2 mL) was added dropwise into a solution of 3a-3d (1 mmol) and Et3N (0.25 mL) in THF (10 mL) under the condition of ice-water bath (5e8 C). After being stirred overnight at room temperature, the solvent was removed under vacuum, water and CH2Cl2 were added. The mixture was extracted with dichloromethane twice and dried over MgSO4. The organic layer was concentrated in vacuum. The crude was refined by chromatography.

## 4.1.3. Synthesis of the intermediates 5e and 5b

A solution of Cyclopentanone (1e, 20 mmol) or cyclohexanone (1b), morpholine (4, 30 mmol) and 4-methylbenzenesulfonic acid (200 mg) in cyclohexane (20 mL) was heated to reflux at 90 C for 4 h. After cooling to room temperature, the mixture was washed with water, dried, and concentrated to obtain the enamine intermediates 5e or 5b as a brown oil, which was directly used for the next reaction.

## 4.1.4. Synthesis of the intermediates 7e and 7b

A mixture of 5e or 5b (10 mmol) and vanillin (10 mmol) was dissolved in ethanol (20 mL) and the resulting solution was stirred at 90 C for 2 h. The residue was concentrated under vacuum and purified by column chromatography to give an orange powder. The powder was re-dissolved in ethanol (10 mL), and 10% HCl solution (4 mL) was added. After being stirred at room temperature for 3 h, distilled water (20 mL) was poured into the reaction flask. A light yellow precipitate of 7e and 7b was collected and washed with water.

## 4.1.5. General procedure for synthesis of compounds 8e1-8e14 and 8b1-8b5

Different aromatic aldehydes (2 mmol) were dissolved into a suspension of the intermediates 7e and 7b (2 mmol) in ethanol (10 mL). Subsequently, HCl gas was bubbled into the mixture for 30 min, and the resulting mixture was stirred at 50e70 C for 2e3 h. After removal of ethanol, water (10 mL) and ethyl ester (20 mL) were poured into the reaction solution. The organic solvent was removed under reduced pressure and the residue was purified by chromatography to obtain the products 8e1-8e14 and 8b1-8b5.

## 4.1.6. General procedure for synthesis of compounds B1eB14

A mixture of compounds 8e1-8e14 (1 mmol) and Et3N (0.25 mL) dissolved in THF (10 mL) was cooled to 0 C. A solution of propionyl chloride or isobutyryl chloride (3 mmol) was added dropwise. The mixture was allowed to warm at room temperature and stirred overnight. After evaporation of the solvent, the mixture was extracted with CH2Cl2, washed with water, and concentrated under vacuum. The desired compounds B1eB14 were eluted with petroleum ether and ethyl acetate by column chromatography, with the final product as various color powders.

4.1.6.1. 2-Methoxy-4-{(E)-{(E)-2-oxo-3-[4-(pyrrolidin-1-yl)benzylidene]cyclopentylidene}me-thyl}phenyl propionate (B7). Orange yellow powder, 6.5% yield, mp: 200.8e202.9 C. HPLC purity: 98.756% (TR <sup>¼</sup> 8.789 min, DW <sup>¼</sup> 420 nm). <sup>1</sup> H-NMR (500 MHz, CDCl3) d: 7.595 (s, 1H, b<sup>0</sup> -H), 7.528 (d, J ¼ 8.5 Hz, 2H, H-2<sup>0</sup> , H-60 ), 7.492 (s, 1H, b-H), 7.207 (d, J ¼ 8.5 Hz, 1H, H-6), 7.171 (s, 1H, H-2), 7.084 (d, J ¼ 8.0 Hz, 1H, H-5), 6.593 (d, J ¼ 8.5 Hz, 2H, H-3<sup>0</sup> , H-50 ), 3.872 (s, 3H, 3-OCH3), 3.370 (t, J ¼ 7.0 Hz, 4H, CH2eNeCH2), 3.082 (s, 4H, 300eCH2, 400eCH2), 2.326 (t, J ¼ 7.5 Hz, 2H, eCOCH2CH3), 2.038 (t, J ¼ 6.5 Hz, 2H, NeCH2eCH2 2), 1.111 (t, <sup>J</sup> <sup>¼</sup> 7.0 Hz, 3H, <sup>e</sup>COCH2CH3). 13C-NMR (125 MHz, CDCl3) d: 195.837, 173.063, 172.415, 151.146, 148.687, 140.369, 138.547, 135.776, 135.198, 133.140, 131.510, 123.034, 122.993, 114.543, 111.842, 55.918, 47.559, 29.688, 26.263, 25.4533, 9.631. HRMS (ESI): calcd for C27H29NO4 [MþH]þ: 432.2175, found: 432.2183.

4.1.6.2. 2-Methoxy-4-{(E)-{(E)-2-oxo-3-[4-(pyrrolidin-1-yl)benzylidene]cyclopentylidene}me-thyl}phenyl isobutyrate (B14). Orange yellow powder, 12.0% yield, mp: 205.8e206.9 C. HPLC purity: 94.308% (TR ¼ 11.258 min, DW ¼ 420 nm). 1H-NMR (500 MHz, CDCl3) d: 7.575 (s, 1H, b<sup>0</sup> -H), 7.512 (d, J ¼ 9.0 Hz, 2H, H-2<sup>0</sup> , H-60 ), 7.488 (s, 1H, <sup>b</sup>-H), 7.204 (d, <sup>J</sup> <sup>¼</sup> 8.0 Hz, 1H, H-6), 7.161 (s, 1H, H-2), 7.069 (d, J ¼ 8.0 Hz, 1H, H-5), 6.697 (d, J ¼ 8.5 Hz, 2H, H-3<sup>0</sup> , H-50 ), 3.860 (s, 3H, 3-OCH3), 3.424 (t, J ¼ 7.0 Hz, 4H, CH2eNeCH2), 3.075 (s, 4H, 300eCH2, 400eCH2), 2.823e2.879 (m, 1H, eCOCH(CH3)2), 2.000e2.020 (m, 4H, NeCH2eCH2 2), 1.211 (d, J ¼ 7.0 Hz, 3H, eCOCH(CH3)2). 13C-NMR (125 MHz, CDCl3) d: 184.589, 169.867, 145.598, 142,875, 134.630, 133.366, 131.531, 129.792, 129.683, 127.811, 125.712, 117.734, 117.243, 109.147, 105.723, 50.640, 38.114, 28.707, 23.520, 23.045, 17.777. HRMS (ESI): calcd for C28H31NO4 [MþH]þ: 446.2330, found: 446.2330.

## 4.1.7. General procedure for synthesis of compounds C1eC5

With the intermediates 8b1-8b5 as starting material, the target compounds C1eC5 were synthesized by applying the same method described for synthesis of B1eB14.

4.1.7.1. 2-Methoxy-4-{(E)-[(E)-3-(4-morpholinobenzylidene)-2 oxocyclohexylidene]methyl}ph-enyl propionate (C1). Yellow powder, 5.4% yield, mp: 186.1e189.4 C. HPLC purity: 96.795% (TR <sup>¼</sup> 8.568 min, DW <sup>¼</sup> 420 nm). <sup>1</sup> H-NMR (500 MHz, CDCl3) d: 7.760 (s, 2H, b-H, b<sup>0</sup> -H), 7.473 (d, J ¼ 8.5 Hz, 2H, H-2<sup>0</sup> , H-60 ), 7.279 (s, 1H, H-2), 7.056e7.073 (m, 2H, H-5, H-6), 6.939 (d, J ¼ 8.5 Hz, 2H, H-3<sup>0</sup> , H-50 ), 3.883 (t, J ¼ 4.5 Hz, 4H, CH2eOeCH2), 3.846 (s, 3H, 3-OCH3), 3.270 (t, J ¼ 3.0 Hz, 4H, CH2eNeCH2), 2.948 (d, J ¼ 2.0 Hz, 4H, 300eCH2, 500eCH2), 2.646 (q, J ¼ 2.5 Hz, 2H, eCOCH2CH3), 1.823 (m, 2H, 400eCH2), 1.297 (t, <sup>J</sup> <sup>¼</sup> 2.5 Hz, 3H, <sup>e</sup>COCH2CH3). 13C-NMR (125 MHz, CDCl3) d: 184.74, 167.17, 145.61, 134.64, 132.15, 131.23, 130.31, 129.65, 127.02, 117.44, 117.37, 109.26, 109.18, 61.36, 50.65, 43.00, 23.37, 23.05, 22.10, 17.69. HRMS (ESI): calcd for C28H31NO5 [MþH]þ: 462.2278, found: 462.2278.

4.1.7.2. 2-Methoxy-4-((E)-((E)-2-oxo-3-(4-(piperidin-1-yl)benzylidene)cyclohexylidene)methyl) phenyl propionate (C2). Orange yellow powder, 19.4% yield, mp: 126.2e129.0 C. HPLC purity: 97.949% (TR <sup>¼</sup> 10.122 min, DW <sup>¼</sup> 420 nm). <sup>1</sup> H-NMR (500 MHz, CDCl3) d: 7.758 (s, 1H, b<sup>0</sup> -H), 7.732 (s, 1H, <sup>b</sup>-H), 7.437 (d, <sup>J</sup> <sup>¼</sup> 9.0 Hz, 2H, H-2<sup>0</sup> , H-60 ), 7.051 (s, 1H, H-2), 7.041 (d, J ¼ 7.5 Hz, 1H, H-6), 7.031 (d, J ¼ 7.5 Hz, 1H, H-5), 6.905 (d, J ¼ 9.0 Hz, 2H, H-3<sup>0</sup> , H-50 ), 3.843 (s, 3H, 3-OCH3), 3.289 (t, J ¼ 6.0 Hz, 4H, CH2eNeCH2), 2.946 (t, J ¼ 6.0 Hz,

c1ccccc1

c1ccccc1

c1ccccc1

c1ccccc1

2H, 300eCH2), 2.905 (t, J ¼ 5.0 Hz, 2H, 500eCH2), 2.627 (q, J ¼ 7.5 Hz, 2H, eCOCH2CH3), 1.802 (t, J ¼ 5.5 Hz, 2H, 400eCH2), 1.693 (t, J ¼ 4.5 Hz, 4H, NeCH2eCH2 2), 1.631 (t, J ¼ 5.0 Hz, 2H, Ne CH2eCH2eCH2), 1.283 (t, <sup>J</sup> <sup>¼</sup> 7.5 Hz, 3H, <sup>e</sup>COCH2CH3). 13C-NMR (125 MHz, CDCl3) d: 189.996, 172.467, 151.749, 150.889, 139.875, 138.032, 136.707, 135.275, 135.048, 132.484, 125.710, 122.701, 122.626, 114.656, 114.466, 55.940, 49.182, 28.712, 28.340, 27.385, 25.501, 24.338, 23.303, 9.148. HRMS (ESI): calcd for C29H33NO4 [MþH]þ: 460.2488, found: 460.2490.

4.1.7.3. 4-{(E)-{(E)-3-[4-(Diethylamino)benzylidene]-2 oxocyclohexylidene}methyl}-2-meth-oxyphenyl propionate (C3). Orange yellow powder, 31.6% yield, mp: 141.7e144.2 C. HPLC purity: 97.569% (TR <sup>¼</sup> 8.019 min, DW <sup>¼</sup> 420 nm). <sup>1</sup> H-NMR (500 MHz, CDCl3) d: 7.776 (s, 1H, b<sup>0</sup> -H), 7.733 (s, 1H, <sup>b</sup>-H), 7.448 (d, <sup>J</sup> <sup>¼</sup> 8.5 Hz, 2H, H-20 , H-60 ), 7.048 (s, 1H, H-2), 7.040 (d, J ¼ 7.5 Hz, 1H, H-6), 7.033 (d, J ¼ 7.5 Hz, 1H, H-5), 6.680 (d, J ¼ 8.5 Hz, 2H, H-3<sup>0</sup> , H-50 ), 3.844 (s, 3H, 3-OCH3), 3.408 (q, J ¼ 7.0 Hz, 4H, CH2eNeCH2), 2.951 (t, J ¼ 5.5 Hz, 2H, 300eCH2), 2.898 (t, J ¼ 5.5 Hz, 2H, 500eCH2), 2.2628 (q, J ¼ 7.5 Hz, 2H, eCOCH2CH3), 1.805 (t, J ¼ 6.0 Hz, 2H, 400eCH2), 1.283 (t, J ¼ 7.5 Hz, 3H, eCOCH2CH3), 1.199 (t, J ¼ 7.0 Hz, 6H, <sup>N</sup>eeCH2eCH3 2). 13C-NMR (125 MHz, CDCl3) <sup>d</sup>: 184.623, 167.213, 145.555, 134.470, 133.447, 131.573, 129.874, 129.670, 127.831, 117.373, 117.300, 109.133, 105.742, 50.638, 39.132, 23.518, 23.042, 22.097, 17.771. HRMS (ESI): calcd for C28H33NO4 [MþH]þ: 448.2488, found: 448.2506.

4.1.7.4. 2-Methoxy-4-{(E)-[(E)-3-(4-morpholinobenzylidene)-2 oxocy-clohexylidene]methyl} phenylisobutyrate (C4). Yellow powder, 8.1% yield, mp: 165.7e168.1 C. HPLC purity: 93.093% (TR <sup>¼</sup> 8.459 min, DW <sup>¼</sup> 420 nm). <sup>1</sup> H-NMR (500 MHz, CDCl3) d: 7.756 (s, 1H, b<sup>0</sup> -H), 7.738 (s, 1H, <sup>b</sup>-H), 7.463 (d, <sup>J</sup> <sup>¼</sup> 9.0 Hz, 2H, H-2<sup>0</sup> , H-60 ), 7.028e7.047 (m, 2H, H-2, H-5), 7.038 (t, 1H, H-6), 6.906 (d, J ¼ 8.5 Hz, 2H, H-3<sup>0</sup> , H-50 ), 3.873 (t, J ¼ 5.0 Hz, 4H, CH2eOeCH2), 3.837 (s, 3H, 3-OCH3), 3.255 (t, J ¼ 5.0 Hz, 4H, CH2eNeCH2), 2.902e2.953 (m, 4H, 300eCH2, 500eCH2), 2.821e2.877 (m, 1H, eCOCH(CH3)2), 1.783e1.830 (m, 2H, 400eCH2), 1.334 (d, J ¼ 7.0 Hz, 6H, eCOCH(CH3)2). 13C-NMR (125 MHz, CDCl3) d: 184.743, 169.843, 145.655, 143.809, 132.116, 131.181, 130.360, 129.563, 128.077, 127.008, 117.446, 117.315, 109.256, 109.197, 61.365, 50.651, 43.000, 28.708, 23.367, 23.050, 17.696, 13.739, 13.718, 13.688. HRMS (ESI): calcd for C29H33NO5 [MþH]þ: 462.2644, found: 462.2646.

4.1.7.5. 2-Methoxy-4-{(E)-{(E)-2-oxo-3-[4-(piperidin-1-yl)benzylidene]cyclohexylidene}methyl} phenyl isobutyrate (C5). Orange yellow powder, 26.7% yield, mp: 158.5e161.6 C. HPLC purity: 93.671% (TR <sup>¼</sup> 12.264 min, DW <sup>¼</sup> 420 nm). <sup>1</sup> H-NMR (500 MHz, CDCl3) d: 7.758 (s, 1H, b<sup>0</sup> -H), 7.733 (s, 1H, <sup>b</sup>-H), 7.437 (d, <sup>J</sup> <sup>¼</sup> 9.0 Hz, 2H, H-2<sup>0</sup> , H-60 ), 7.042e7.063 (m, 2H, H-2, H-5), 7.026 (s, 1H, H-6), 6.906 (d, J ¼ 8.5 Hz, 2H, H-3<sup>0</sup> , H-50 ), 3.834 (s, 3H, 3-OCH3), 3.288 (t, J ¼ 5.5 Hz, 4H, CH2eNeCH2), 2.944 (t, J ¼ 5.5 Hz, 2H, 300eCH2), 2.902 (t, J ¼ 5.5 Hz, 2H, 500eCH2), 2.819e2.875 (m, 1H, eCOCH(CH3)2), 1.801 (t, J ¼ 6.0 Hz, 2H, 400eCH2), 1.684e1.693 (m, 4H, NeCH2eCH2 2), 1.626 (d, J ¼ 4.5 Hz, 2H, NeCH2eCH2eCH2), 1.333 (d, J ¼ 7.0 Hz, 6H, eCOCH(CH3)2). 13C-NMR (125 MHz, CDCl3) d: 184.685, 169.844, 146.441, 145.627, 134.726, 132.693, 131.350, 130.031, 129.673, 127.192, 120.407, 117.416, 117.280, 109.356, 109.171, 50.646, 43.885, 28.710, 23.430, 23.057, 20.213, 19.050, 17.733, 13.750, 13.696. HRMS (ESI): calcd for C30H35NO4 [MþH]þ: 474.2644, found: 474.2654.

## 4.1.8. General procedure for synthesis of compounds D1-D13

The preparation procedure of compounds D1-D13 was the same as compounds 8e1-8e14, which used the different substituted aromatic aldehyde for the aldol condensation reaction.

4.1.8.1. (2E,6E)-2-(4-hydroxy-3-methoxybenzylidene)-6-(4-(pyrrolidi-1-yl)benzylidene)cyclo-hexanone (D13). Orange yellow powder, 15.2% yield, mp: 203.2e206.1 C. HPLC purity: 99.267% (TR <sup>¼</sup> 6.230 min, DW <sup>¼</sup> 450 nm). <sup>1</sup> H-NMR (500 MHz, DMSO-d6) d: 7.558 (d, J ¼ 7.5 Hz, 2H, H-2<sup>0</sup> , H-60 ), 7.436 (s, 1H, b<sup>0</sup> -H), 7.419 (s, 1H, b-H), 7.100 (s, 1H, H-2), 7.013 (d, J ¼ 8.5 Hz, 1H, H-6), 6.851 (d, J ¼ 7.0 Hz, 1H, H-5), 6.595 (d, J ¼ 8.5 Hz, 2H, H-3<sup>0</sup> , H-50 ), 3.817 (s, 3H, 3-OCH3), 3.292 (t, J ¼ 6.5 Hz, 4H, CH2eNeCH2), 2.874 (d, J ¼ 5.0 Hz, 4H, 300eCH2, 500eCH2), 1.968 (t, J ¼ 6.5 Hz, 4H, NeCH2eCH2 2), 1.728 (t, <sup>J</sup> <sup>¼</sup> 5.5 Hz, 2H, 400eCH2). 13C-NMR (125 MHz, DMSO-d6) <sup>d</sup>: 188.028, 147.901, 147.612, 147.388, 137.017, 135.397, 133.754, 132.536, 130.769, 127.043, 124.032, 122.370, 115.490, 114.655, 111.547, 55.617, 47.171, 28.229, 27.919, 24.919, 22.631. HRMS (ESI): calcd for C25H27NO3 [MþH]þ: 390.2069, found: 390.2078.

## 4.1.9. Preparation of the protected vanillin 10

A solution of Vanillin (2, 10 mmol), 3,4-2H-dihydropyran (9, 40 mmol) and pyridinium 4-toluenesulfonate (2.51 mmol) in CH2Cl2 (30 mL) was stirred at room temperature for 3 h. The resulting mixture was extracted with CH2Cl2 twice and concentrated in vacuum. The protected product 10 was purified by silica gel column chromatography as colorless oil.

## 4.1.10. Preparation of the unilateral substituted intermediate 11

To a solution of compound 10 (5 mmol) in acetone (20 mL) a solution of 20% NaOH (1 mL) was added dropwise slowly at room temperature. TLC monitored the endpoint and removed the solvent. The residue was extracted with ethyl ester (20 mL) and washed water (10 mL) twice. After evaporating the ethyl ester, the mixture was purified by column chromatography to obtain the intermediate 11 as white powder.

## 4.1.11. General procedure for synthesis of compounds D14-D15

To a mixture of dry powdered reactant 10 (1 mmol) and corresponding substituted benzaldehyde (1 mmol), a suspension of NaOH in dioxane (5g NaOH dispersed in 20 mL dioxane) was added dropwise until the reaction color turned wine red. The solution was stirred at room temperature for 4 h. The residue was concentrated and extracted with ethyl ester (20 mL). After purification by column chromatography, the colored powder was dissolved in ethanol (10 mL), and de-protected by 10% HCl solution (4 mL). The formed precipitates was filtered and purified by column chromatography to give the desired products D14-D15.

4.1.11.1. (1E,4E)-1-(4-Hydroxy-3-methoxyphenyl)-5-[4-(pyrrolidin-1-yl)phenyl]penta-1,4-dien-3-one (D14). Red powder, 3.0% yield, mp: 204.3e207.4 C. HPLC purity: 97.934% (TR ¼ 7.402 min, DW <sup>¼</sup> 450 nm). <sup>1</sup> H-NMR (500 MHz, DMSO-d6) <sup>d</sup>: 7.559 (d, <sup>J</sup> <sup>¼</sup> 7.0 Hz, 2H, H-20 , H-60 ), 7.547 (d, <sup>J</sup> <sup>¼</sup> 15.5 Hz, 1H, <sup>b</sup><sup>0</sup> -H), 7.539 (d, J ¼ 15.5 Hz, 1H, <sup>b</sup>-H), 7.148 (s, 1H, H-2), 7.059 (d, <sup>J</sup> <sup>¼</sup> 8.0 Hz, 1H, H-6), 6.960 (d, <sup>J</sup> <sup>¼</sup> 16.0 Hz, 1H, <sup>a</sup><sup>0</sup> -H), 6.829 (d, <sup>J</sup> <sup>¼</sup> 15.0 Hz, 1H, <sup>a</sup>-H), 6.579 (d, J ¼ 8.0 Hz, 2H, H-3<sup>0</sup> , H-50 ), 6.504 (d, J ¼ 7.0 Hz, 1H, H-5), 3.769 (s, 3H, 3-OCH3), 3.308 (d, J ¼ 6.5 Hz, 4H, CH2eNeCH2), 1.974 (s, 4H, <sup>N</sup>eCH2eCH2 2). 13C-NMR (125 MHz, DMSO-d6) <sup>d</sup>: 187.574, 149.248, 149.199, 147.938, 143.235, 141.853, 130.341, 130.196, 126.513, 123.319, 123.078, 121.503, 120.195, 115.646, 111.762, 111.450, 55.723, 47.423, 24.911. HRMS (ESI): calcd for C22H23NO3 [MþH]þ: 350.1756, found: 350.1760.

4.1.11.2. (1E,4E)-1-[4-(Diethylamino)phenyl]-5-(4-hydroxy-3 methoxyphenyl)penta-1,4-dien-3-one (D15). Red powder, 7.8% yield, mp: 134.7e135 C. HPLC purity: 99.900% (TR ¼ 6.311 min, DW <sup>¼</sup> 450 nm). <sup>1</sup> H-NMR (500 MHz, DMSO-d6) d: 7.624 (d, <sup>J</sup> <sup>¼</sup> 15.5 Hz, 1H, <sup>b</sup><sup>0</sup> -H), 7.592 (d, <sup>J</sup> <sup>¼</sup> 15.5 Hz, 1H, <sup>b</sup>-H), 7.567 (d, , H-60 ), 7.352 (s, 1H, H-2), 7.182 (d, J ¼ 8.0 Hz, 1H,

c1ccccc1

c1ccccc1

H-6), 7.119 (d, <sup>J</sup> <sup>¼</sup> 15.5 Hz, 1H, <sup>a</sup><sup>0</sup> -H), 6.977 (d, <sup>J</sup> <sup>¼</sup> 15.5 Hz, 1H, <sup>a</sup>-H), 6.812 (d, J ¼ 8.0 Hz, 1H, H-5), 6.709 (d, J ¼ 8.0 Hz, 2H, H-3<sup>0</sup> , H-50 ), 3.848 (s, 3H, 3-OCH3), 3.414 (d, J ¼ 6.5 Hz, 4H, CH2eNeCH2), 1.124 (t, <sup>J</sup> <sup>¼</sup> 6.5 Hz, 6H, NeCH2eCH3 2). 13C-NMR (125 MHz, DMSO-d6) d: 187.586, 149.264, 149.216, 147.942, 143.006, 141.864, 130.527, 126.502, 123.295, 123.084, 121.242, 120.244, 115.651, 111.472, 111.224, 55.727, 43.759, 12.437. HRMS (ESI): calcd for C22H25NO3 [MþH]þ: 352.1912, found: 352.1928.

#### 4.2. In vitro kinase assay

Based on microfluidics chip technology, the kinase activity of FGFR1 was determined by Mobility Shift Assay on EZ Reader (Caliper Life Sciences, MA). As the experimental condition was constrained, the kinase activity screening and ATP competitive assay both were authorized to ChemPartner (Shanghai, China). The specific protocol is described briefly as follows:

The enzyme solution was prepared in 1.25 kinase base buffer (62.5 mM HEPES, 0.001875% Brij-35, 12.5 mM MgCl2, 2.5 mM DTT), stop buffer (100 mM HEPES, 0.015% Brij-35, 0.2% coating reagent No. 3, 50 mM EDTA) The compound was dissolved in DMSO, and diluted to the specific concentration with water in 384-well plate. <sup>5</sup> <sup>m</sup>L of compound solution was diluted with 10 <sup>m</sup>L of 2.5 enzyme solution. 10 mM EDTA was used as the blank control. The mixture was incubated at room temperature for 10 min. 10 mL of peptide solution (2.5, FAM-P22, add FAM-labeled peptide and ATP in the 1.25 kinase base buffer) was added to each well of the 384-well assay plate. After incubation at 28 C for 1 h, 25 mL of stop buffer was added into the mixture to stop the reaction. Conversion data were collected on Caliper EZ reader (Hopkinton, MA), the percent inhibition of kinase activity was calculated based on the formula below:

## Inhibition % ¼ (max conversion)/(max min) 100%,

"Max" stands for DMSO control; "Min" stands for low control.

To determine the half-maximal inhibitory concentration of the compounds relative to kinase activity, 10 gradient concentrations of compounds (100, 33.33, 11.11, 3.70, 1.23, 0.41, 0.14, 0.046, 0.015, and 0.005 mM) were set up. The inhibition ratios of different concentrations relative to kinase were calculated and the concentrationinhibition rate curve was fitted by GraphPad Prism.

#### 4.3. Quantitative structure-activity relationships analysis

#### 4.3.1. Materials and descriptors selection

All products were drawn and saved as SDF format. The compounds were calculated by the ChemoPy descriptor calculation program [\[37,38\].](#page-14-0) After calculation of molecular descriptors, about 1000 molecular descriptors based on molecular structure were obtained. In this study, the descriptors include the constitutional descriptors, physicochemical descriptors, topological descriptors, geometrical descriptors, charge (electronic) descriptors, etc. After calculation of the molecular descriptors, those that were constant descriptors for all molecules were eliminated and pairs of variables with a correlation coefficient greater than 0.75 were classified as inter-correlated and one in each correlated pair was deleted.

#### 4.3.2. Multiple linear regression (MLR) analysis

Multiple linear regression (MLR) analysis is a statistical technique that uses several explanatory variables to predict the outcome of a response variable. The goal of MLR is to model the relationship between the explanatory and response variables. In our present study, MLR was performed using R program, a widely used tool for statistical computing and graphics, to derive QSAR models. The biological data used in this study were the FGFR1 inhibitory rates compared with negative group. Compounds with negative values were abandoned because of their anti-cancer activities. The inhibition rates against FGFR1 were used as dependent variables in the linearization procedure. Subsequently, stepwise Multiple Linear Regression (Stepwise-MLR) was used to select the significant descriptors. The most relevant descriptors were used as independent variables.

#### 4.3.3. Validation of the models

To confirm that the lineal model was reliable, testing the predictive ability and generalizing the methods by cross-validation is required. The leave-one-out (LOO) procedure was employed. When a data point was removed from the analyzed set, the regression was recalculated and the predicted value for that point was compared to its actual value. This process was repeated until each datum had been omitted once and the sum of squares of these deletion residuals could be used to calculate q2 , an equivalent statistic to R<sup>2</sup> .

#### 4.4. ATP competitive inhibition assay

With the use of kinase inhibitory activity screening assay, four gradient concentrations of compounds to be measured were set up. At each concentration of different compounds, eight concentrations of ATP (4192, 2096, 1048, 524, 262, 131, 66, and 33 mM) were established to determine the conversion of substrate peptide catalyzed by FGFR1 kinase within 1 h. The inhibition curve was fitted by GraphPad Prism.

#### 4.5. Molecular docking

The binding pose of compound D12 and D15 in FGFR1's binding site was predicted by the software Autodock (version 4.2.6) [\[39\].](#page-14-0) The crystallographic co-ordinates for human wild type FGFR1 (PDB ID: 3RHX) was retrieved from the Protein Data Bank (PDB). Prior to docking, protein structure was prepared by removing water molecules and other ligands using PyMol software [\[40\]](#page-14-0). Individual ligand files were prepared for docking using the prepare\_ligand4.py script, then charges and hydrogen atoms of FGFR1 protein were added using the prepare\_receptor4.py script from last version of AutoDockTools 1.5.6. A grid box size of 60 60 60 dimensions, with a spacing of 0.375 Å between the grid points, was implemented and covered almost the entire FGFR1 binding site. The grid parameter files were created by setting up of map files directly. The Lamarckian genetic algorithm (LGA) was applied to deal with the protein and inhibitors interactions. The docking parameters were as follows: trials of 100 dockings, the number of individuals in population was set to 300, the crossover rate was 0.8, and the local search rate was 0.06 and 25 million of maximum number of energy evaluations. Other settings were set to default. AutoDockTools and PyMol was used to analyze the docking results.

#### 4.5.1. Molecular dynamics (MD) simulations

Molecular dynamics (MD) simulations were performed to verify whether the docking results are stable enough. Molecular dynamics (MD) simulations of D12/FGFR1 complex system and D15/FGFR1 system were carried out. The initial complexes came from the top ranked binding pose, cluster by the docking results with the root mean square deviations (RMSD ¼ 2) of each pose. The partial charges of the D12 and D15 were derived by using the restrained electrostatic potential (RESP) fitting procedure based on the electrostatic potentials calculated by Hartree-Fock (HF) method with 6- 31G (d) basis set in the Gaussian09 Package [\[41\]](#page-14-0). Molecular mechanics parameters, from the ff99SB and GAFF force fields using the LEaP module of AMBER 11 software packages, were assigned to the

c1ccccc1

c1ccccc1

protein and the ligand respectively [\[42,43\].](#page-14-0) All the water molecules and the specified ligand were deleted. The two systems were all solvated in a box of TIP3P water molecules with a hydration shell of 10 Å. Additionally, an appropriate number of sodium ions was used

## to neutralize these two systems.

The AMBER 11 software package was employed, using the same protocol, for all simulations [\[42\]](#page-14-0). We carried out an equilibration protocol consisting of an initial minimization of water box of 5000 steps, 2500 steps for the steepest descent and 2500 steps in the conjugate gradient. Then, the TIP3P water box was heated at constant volume until 300 K using a time constant for the heat bath with a coupling time of 100 ps? Equilibration was at 300 K under constant pressure of 100 ps? Before the production of MD, the whole system was equilibrated at 100 ps at a constant pressure of 1 bar, and the Langevin temperature scaling was turned on with a collision frequency of 2 ps [\[44\].](#page-14-0) The last step used 50 ns of MD simulation without any restraints. In addition, short-range nonbonded interactions were calculated based on a cutoff of 8.0 Å. Periodic Boundary Conditions were turned on in every step of the whole process. Long-range electrostatic interactions were handled by the Particle Mesh Ewald (PME) algorithm [\[45\]](#page-14-0). The SHAKE method was used to constrain hydrogen atoms and the time step was set to 2 fs [\[46\]](#page-14-0). The coordinates were saved every 10 ps for the subsequent analysis.

#### 4.5.2. Binding free energy calculations and decomposition analysis

The binding free energies were determined by the molecular mechanics/generalized Born surface area (MM-GBSA) method as implemented in AMBER 11. As to each system, 500 snapshots from the last 10 ns MD trajectory were chosen for the calculations. The binding free energy was calculated according to the equation:

$$
\Delta \mathbf{G}\_{\text{bind}} = \Delta \mathbf{G}\_{\text{complex}} - \left(\Delta \mathbf{G}\_{\text{receeptor}} + \Delta \mathbf{G}\_{\text{ligand}}\right) \tag{1}
$$

$$
\Delta G\_{\text{bind}} = \Delta \mathbf{H} - \mathbf{T} \Delta \mathbf{S} \approx \Delta \mathbf{E}\_{\text{gas}} + \Delta \mathbf{G}\_{\text{sol}} - \mathbf{T} \Delta \mathbf{S} \tag{2}
$$

$$
\Delta \mathbf{G}\_{\text{gas}} = \Delta \mathbf{G}\_{\text{VDW}} + \Delta \mathbf{G}\_{\text{elec}} \tag{3}
$$

$$
\Delta \mathbf{G}\_{\text{sol}} = \Delta \mathbf{G}\_{\text{GB}} + \Delta \mathbf{G}\_{\text{SA}} \tag{4}
$$

In the equation (3), △Ggas between protein and ligand is the gas-phase interaction, including the van der Waals and the electrostatic energies; △GGB and △GSA are the polar and nonpolar components of the desolvation free energy, respectively; T△S, at temperature T, represents the change of conformational entropy upon ligand binding.

#### 4.6. Biological activity experiments

#### 4.6.1. Cell culture

SGC7901 was obtained from Cell Bank of Chinese Academy of Sciences (Wuhan, China), and MGC823, BGC803 were bought from Shanghai Institute of Biosciences and Cell Resources Center (Chinese Academy of Sciences, Shanghai, China). All cells were grown in RMPI-1640 media (Gibco) supplemented with 10% FBS (Gibco), 1% Penicilin Streptomycin (Gibco) in a humidified ThermoForma (Thermo Fisher Scientific) containing 5% CO2 at 37 C.

## 4.6.2. MTT cytotoxicity assay

Cells (4 103 ) were seeded in a 96-well cell culture plate and treated with compounds for 72 h. Assay was made by adding 20 mL of methylthiazoletetrazolium (MTT; 0.5 mg/mL; M5655; Sigma) to the culture medium and incubated at 37 C for 4 h. At the end of incubation, the produced crystal was dissolved in 150 mL dimethyl

## sulfoxide (sigma) and quantified at 490 nm.

#### 4.6.3. Western blot analysis

Treated cells were washed with PBS and harvested using RIPA Buffer (AR0103; BOSTER, Wuhan, China) with 1% PMSF (100 mM; ST506; Beyotime). Protein quantification was performed using the Bradford 1 Dye Reagent (#5000205; Bio-Rad). Cell extracts were separated by 10% SDS-PAGE and transferred to PVDF membrane (Millipore). Then the membrane was blocked for 90 min at room temperature in TBS with 0.1% Tween and 5% milk powder. The membrane was incubated with the following primary antibodies: Phospho-Flg (Tyr653/654; 1:300; sc-30262; Santa cruz Biotechnology, lnc.), p-ERK (1:300; sc-7383; Santa cruz Biotechnology, lnc.), cleaved PARP-1(194C1439; 1:300; sc-56196; Santa cruz Biotechnology, lnc.), cyclin D1 (1:300; sc-753 Santa cruz Biotechnology, lnc.), Bcl-2 (1:300; sc-492; Santa cruz Biotechnology, lnc.), or GAPDH (1:300; sc-25778; Santa cruz Biotechnology, lnc.). After washing, the membranes were incubated with the relevant secondary antibodies, visualization of bands was by enhanced chemiluminescence (ECL; Bio-Rad), and densitometric analysis made using Image J software.

#### 4.6.4. Hoechst 33342 staining

SGC7901 cells at logarithmic growth were seeded in 6-well cell culture plate and treated with DMSO at different concentrations of D12 or D15 or 10 mM A117 for 12 h. Following fixation with 4% paraformaldelyde for 10 min at room temperature (RT), cells were washed and stained with Hoechst 33342 (Beyotime) for 10 min. Cells were observed under a fluorescence microscope (Nikon, Tokyo, Japan) using appropriate filters for blue fluorescence.

#### 4.6.5. Cell cycle analysis

Cells (3 <sup>10</sup><sup>5</sup> ) were treated with agents for 8 h, washed with PBS, collected, and fixed in 75% ice-cold ethanol for 4 h. After washing with PBS, the cells were stained with 500 mL of propidium iodide (PI) (Becton, Dickinson and Company) for 10 min at 4 C in the dark. The cell suspension was filtered with 200 - mesh gauze and subjected to a FACSCalibur instrument (BD Biosciences Clontech). Data were analyzed using the ModFit DNA analysis program.

#### 4.6.6. Statistical analysis

The results were presented as the mean ± standard error (SEMs). The statistics were performed using Student's t-test in GraphPad Pro (GraphPad, San Diego, CA, USA). A value of P less than 0.05 (p < 0.05) was considered statistically significant. All experiments were repeated a minimum of three times.

## Conflict of interest

The authors have declared no conflict of interest.

#### Acknowledgment

This work was supported by the National Natural Science Foundation of China (Grant Nos. 81402839, 81272462, 81473242), the Natural Science Foundation of Zhejiang Province of China (LY17H160059), and the Technology Foundation for Medical Science of Zhejiang Province (Grant No. 2012KYA129).

#### Appendix A. Supplementary data

Supplementary data associated with this article can be found in the online version, at [http://dx.doi.org/10.1016/j.ejmech.2016.10.](http://dx.doi.org/10.1016/j.ejmech.2016.10.066) [066](http://dx.doi.org/10.1016/j.ejmech.2016.10.066). These data include MOL files and InChiKeys of the most important compounds described in this article.

c1ccccc1

c1ccccc1

c1ccccc1

c1ccccc1

#### <span id="page-14-0"></span>References

- [1] [J. Ferlay, I. Soerjomataram, R. Dikshit, S. Eser, C. Mathers, M. Rebelo,](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref1) [D.M. Parkin, D. Forman, F. Bray, Cancer incidence and mortality worldwide:](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref1) [sources, methods and major patterns in GLOBOCAN 2012, Int. J. Cancer 136](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref1) [\(2015\) E359](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref1)e[E386.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref1)
- [2] [R.L. Siegel, K.D. Miller, A. Jemal, Cancer statistics, 2015, CA-A Cancer J. Clin. 65](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref2) [\(2015\) 5](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref2)e[29](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref2).
- [3] [T. Oshima, M. Masuda, Molecular targeted agents for gastric and gastro](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref3)[esophageal junction cancer, Surg. Today 42 \(2012\) 313](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref3)e[327.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref3)
- [4] [J. Wang, R. Xu, J. Li, Y. Bai, T. Liu, S. Jiao, G. Dai, J. Xu, Y. Liu, N. Fan, Y. Shu, Y. Ba,](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref4) [D. Ma, S. Qin, L. Zheng, W. Chen, L. Shen, Randomized multicenter phase III](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref4) study of a modifi[ed docetaxel and cisplatin plus](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref4) fluorouracil regimen compared with cisplatin and fluorouracil as fi[rst-line therapy for advanced or](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref4) [locally recurrent gastric cancer, Gastric Cancer 19 \(2016\) 234](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref4)e[244](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref4).
- [5] [S.S. Khakoo, A. Georgiou, J.S. Waters, A restrospective analysis of toxicities](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref5) [encountered with palliative epirubicin, oxaliplatin, and capecitabine \(EOX\)](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref5) [chemotherapy for advanced esophagogastric cancer, J. Clin. Oncol. 32 \(2014\)](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref5).
- [6] [E.C. Smyth, D. Cunningham, Targeted therapy for gastric cancer, Curr. Treat.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref6) [Options Oncol. 13 \(2012\) 377](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref6)e[389](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref6).
- [7] [H. Arai, S. Hironaka, T. Suzuki, K. Sudo, K. Nakamura, K. Minashi, T. Hara,](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref7) [T. Denda, T. Yamaguchi, A retrospective study of early toxicity of weekly](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref7) [paclitaxel as second line chemotherapy for advanced gastric cancer, Ann.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref7) [Oncol. 26 \(2015\), 132](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref7)e[132.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref7)
- [8] [T. Aoyama, T. Yoshikawa, Targeted therapy: apatinib new third-line option](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref8) [for refractory gastric or GEJ cancer, Nat. Rev. Clin. Oncol. 13 \(2016\) 268](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref8)e[270](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref8).
- [9] [V. Brower, Apatinib in treatment of refractory gastric cancer, Lancet Oncol. 17](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref9) [\(2016\) e137](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref9).
- [10] [J. Zhang, P.L. Yang, N.S. Gray, Targeting cancer with small molecule kinase](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref10) [inhibitors, Nat. Rev. Cancer 9 \(2009\) 28](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref10)e[39](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref10).
- [11] [C. Hierro, J. Rodon, J. Tabernero, Fibroblast growth factor \(FGF\) receptor/FGF](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref11) [inhibitors: novel targets and strategies for optimization of response of solid](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref11) [tumors, Semin. Oncol. 42 \(2015\) 801](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref11)e[819.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref11)
- [12] [M.H. Schafer, P. Lingohr, A. Strasser, N.C. Lehnen, M. Braun, S. Perner, T. Holler,](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref12) [G. Kristiansen, J.C. Kalff, I. Gutgemann, Fibroblast growth factor receptor 1](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref12) gene amplifi[cation in gastric adenocarcinoma, Hum. Pathol. 46 \(2015\)](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref12) [1488](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref12)e[1495](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref12).
- [13] [M.V. Dieci, M. Arnedos, F. Andre, J.C. Soria, Fibroblast growth factor receptor](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref13) [inhibitors as a cancer treatment: from a biologic rationale to medical per](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref13)[spectives, Cancer Discov. 3 \(2013\) 264](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref13)e[279.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref13)
- [14] [M. Touat, E. Ileana, S. Postel-Vinay, F. Andre, J.C. Soria, Targeting FGFR](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref14) [signaling in Cancer, Clin. Cancer Res. 21 \(2015\) 2684](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref14)e[2694.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref14)
- [15] [E.P. Carter, A.E. Fearon, R.P. Grose, Careless talk costs lives:](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref15) fibroblast growth [factor receptor signalling and the consequences of pathway malfunction,](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref15) [Trends Cell Biol. 25 \(2015\) 221](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref15)e[233](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref15).
- [16] [G. Liang, Z. Liu, J. Wu, Y. Cai, X. Li, Anticancer molecules targeting](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref16) fibroblast [growth factor receptors, Trends Pharmacol. Sci. 33 \(2012\) 531](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref16)e[541](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref16).
- [17] [A. Dutt, A.H. Ramos, P.S. Hammerman, C. Mermel, J. Cho, T. Sharifnia,](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref17) [A. Chande, K.E. Tanaka, N. Stransky, H. Greulich, N.S. Gray, M. Meyerson, In](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref17)hibitor-sensitive FGFR1 amplifi[cation in human non-small cell lung cancer,](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref17) [PLoS One 6 \(2011\) e20351](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref17).
- [18] [I. Tsimafeyeu, L. Demidov, E. Stepanova, N. Wynn, H. Ta, Overexpression of](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref18) fi[broblast growth factor receptors FGFR1 and FGFR2 in renal cell carcinoma,](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref18) [Scand. J. Urol. Nephrol. 45 \(2011\) 190](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref18)e[195.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref18)
- [19] [V.K. Jain, N.C. Turner, Challenges and opportunities in the targeting of](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref19) fibro[blast growth factor receptors in breast cancer, Breast Cancer Res. 14 \(2012\)](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref19) [208](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref19).
- [20] [N. Turner, R. Grose, Fibroblast growth factor signalling: from development to](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref20) [cancer, Nat. Rev. Cancer 10 \(2010\) 116](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref20)e[129](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref20).
- [21] [G. Daniele, J. Corral, L.R. Molife, J.S. de Bono, FGF receptor inhibitors: role in](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref21) [cancer therapy, Curr. Oncol. Rep. 14 \(2012\) 111](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref21)e[119](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref21).
- [22] [N. Miyamoto, H. Yamamoto, C. Miyamoto, T. Maehata, K. Nosho, H. Taniguchi,](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref22) [K. Yamashita, Y. Adachi, Y. Arimura, F. Itoh, K. Imai, Y. Shinomura, Over](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref22)[expression of the receptor tyrosine kinase epha4 plays an important role in](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref22) [the progression of human gastric cancers, Gastroenterology 134 \(2008\)](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref22) [A611](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref22)e[A612](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref22).
- [23] [D. Wen, S. Li, F. Ji, H. Cao, W. Jiang, J. Zhu, X. Fang, miR-133b acts as a tumor](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref23) [suppressor and negatively regulates FGFR1 in gastric cancer, Tumor Biol. 34](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref23) [\(2013\) 793](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref23)e[803.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref23)
- [24] [L. Xie, X. Su, L. Zhang, X. Yin, L. Tang, X. Zhang, Y. Xu, Z. Gao, K. Liu, M. Zhou,](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref24) [B. Gao, D. Shen, L. Zhang, J. Ji, P.R. Gavine, J. Zhang, E. Kilgour, X. Zhang, Q. Ji,](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref24)

FGFR2 gene amplifi[cation in gastric cancer predicts sensitivity to the selective](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref24) [FGFR inhibitor AZD4547, Clin. Cancer Res. 19 \(2013\) 2572](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref24)e[2583](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref24).

- [25] [K. Schmidt, C. Moser, C. Hellerbrand, D. Zieker, C. Wagner, J. Redekopf,](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref25) [H.J. Schlitt, E.K. Geissler, S.A. Lang, Targeting](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref25) fibroblast growth factor receptor [\(FGFR\) with BGJ398 in a gastric Cancer model, Anticancer Res. 35 \(2015\)](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref25) [6655](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref25)e[6665](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref25).
- [26] [Z. Zhao, H. Wu, L. Wang, Y. Liu, S. Knapp, Q. Liu, N.S. Gray, Exploration of type](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref26) [II binding mode: a privileged approach for kinase inhibitor focused drug](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref26) [discovery? ACS Chem. Biol. 9 \(2014\) 1230](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref26)e[1241](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref26).
- [27] [R.A. Norman, D. Toader, A.D. Ferguson, Structural approaches to obtain kinase](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref27) [selectivity, Trends Pharmacol. Sci. 33 \(2012\) 273](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref27)e[278.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref27)
- [28] [S. Eathiraj, R. Palma, M. Hirschi, E. Volckova, E. Nakuci, J. Castro, C.R. Chen,](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref28) [T.C. Chan, D.S. France, M.A. Ashwell, A novel mode of protein kinase inhibition](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref28) [exploiting hydrophobic motifs of autoinhibited kinases: discovery of ATP](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref28)independent inhibitors of fi[broblast growth factor receptor, J. Biol. Chem.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref28) [286 \(2011\) 20677](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref28)e[20687.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref28)
- [29] [Y. Wang, Y. Cai, J. Ji, Z. Liu, C. Zhao, Y. Zhao, T. Wei, X. Shen, X. Zhang, X. Li,](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref29) G. Liang, Discovery and identifi[cation of new non-ATP competitive FGFR1](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref29) [inhibitors with therapeutic potential on non-small-cell lung cancer, Cancer](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref29) [Lett. 344 \(2014\) 82](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref29)e[89](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref29).
- [30] [Y. Zhang, L. Zhao, J. Wu, X. Jiang, L. Dong, F. Xu, P. Zou, Y. Dai, X. Shan, S. Yang,](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref30) [G. Liang, Synthesis and evaluation of a series of novel asymmetrical curcumin](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref30) analogs for the treatment of infl[ammation, Molecules 19 \(2014\) 7287](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref30)e[7307](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref30).
- [31] [Y.L. Zhang, X. Jiang, K.S. Peng, C.W. Chen, L.L. Fu, Z. Wang, J.P. Feng, Z.G. Liu,](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref31) [H.J. Zhang, G. Liang, Z. Pan, Discovery and evaluation of novel anti](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref31)infl[ammatory derivatives of natural bioactive curcumin, Drug Des. Dev.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref31) [Ther. 8 \(2014\) 2161](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref31)e[2171.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref31)
- [32] [L.L. Dong, S.Q. Zheng, Y.L. Zhang, X. Jiang, J.Z. Wu, X.Q. Zhang, X.O. Shan,](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref32) [D.D. Liang, S.L. Ying, J.P. Feng, G. Liang, Design, synthesis, and evaluation of](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref32) [semi-conservative mono-carbonyl analogs of curcumin as anti-in](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref32)flammatory [agents against lipopolysaccharide-induced acute lung injury, Med. Chem.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref32) [Commun. 6 \(2015\) 1544](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref32)e[1553.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref32)
- [33] [L.N. Johnson, Protein kinase inhibitors: contributions from structure to clinical](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref33) [compounds, Q. Rev. Biophys. 42 \(2009\) 1](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref33)e[40](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref33).
- [34] [J.L. Qu, X.J. Qu, M.F. Zhao, Y.E. Teng, Y. Zhang, K.Z. Hou, Y.H. Jiang, X.H. Yang,](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref34) [Y.P. Liu, Gastric cancer exosomes promote tumour cell proliferation through](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref34) [PI3K/Akt and MAPK/ERK activation, Dig. Liver Dis. 41 \(2009\) 875](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref34)e[880.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref34)
- [35] [M. Fu, C. Wang, Z. Li, T. Sakamaki, R.G. Pestell, Minireview: cyclin D1: normal](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref35) [and abnormal functions, Endocrinology 145 \(2004\) 5439](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref35)e[5447.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref35)
- [36] [W.K. Wu, C.H. Cho, C.W. Lee, D. Fan, K. Wu, J. Yu, J.J. Sung, Dysregulation of](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref36) [cellular signaling in gastric cancer, Cancer Lett. 295 \(2010\) 144](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref36)e[153](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref36).
- [37] [D.S. Cao, S. Liu, L. Fan, Y.Z. Liang, QSAR analysis of the effects of OATP1B1](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref37) [transporter by structurally diverse natural products using a particle swarm](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref37) [optimization-combined multiple linear regression approach, Chemom. Intell.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref37) [Lab. Syst. 130 \(2014\) 84](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref37)e[90](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref37).
- [38] [D.S. Cao, Q.S. Xu, Q.N. Hu, Y.Z. Liang, ChemoPy: freely available python](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref38) [package for computational biology and chemoinformatics, Bioinformatics 29](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref38) [\(2013\) 1092](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref38)e[1094](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref38).
- [39] [G.M. Morris, R. Huey, W. Lindstrom, M.F. Sanner, R.K. Belew, D.S. Goodsell,](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref39) [A.J. Olson, AutoDock4 and AutoDockTools4: automated docking with selective](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref39) receptor fl[exibility, J. Comput. Chem. 30 \(2009\) 2785](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref39)e[2791.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref39)
- [40] [W.L. DeLano, PyMOL molecular viewer: updates and re](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref40)finements, Abstr. Pap. [Am. Chem. Soc. 238 \(2009\).](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref40)
- [41] [M. Frisch, G. Trucks, H.B. Schlegel, G. Scuseria, M. Robb, J. Cheeseman,](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref41) [G. Scalmani, V. Barone, B. Mennucci, G.e. Petersson, Gaussian 09, Gaussian, Inc,](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref41) [Wallingford, CT, 2009](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref41).
- [42] [D.A. Pearlman, D.A. Case, J.W. Caldwell, W.S. Ross, T.E. Cheatham, S. DeBolt,](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref42) [D. Ferguson, G. Seibel, P. Kollman, AMBER, a package of computer programs](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref42) [for applying molecular mechanics, normal mode analysis, molecular dynamics](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref42) [and free energy calculations to simulate the structural and energetic prop](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref42)[erties of molecules, Comput. Phys. Commun. 91 \(1995\) 1](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref42)e[41.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref42)
- [43] [J. Wang, R.M. Wolf, J.W. Caldwell, P.A. Kollman, D.A. Case, Development and](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref43) testing of a general amber force fi[eld, J. Comput. Chem. 25 \(2004\) 1157](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref43)e[1174](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref43).
- [44] [J.A. Izaguirre, D.P. Catarello, J.M. Wozniak, R.D. Skeel, Langevin stabilization of](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref44) [molecular dynamics, J. Chem. Phys. 114 \(2001\) 2090](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref44)e[2098.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref44)
- [45] [C. Sagui, T.A. Darden, Molecular dynamics simulations of biomolecules: long](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref45)[range electrostatic effects, Annu. Rev. Biophys. Biomol. Struct. 28 \(1999\)](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref45) [155](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref45)e[179.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref45)
- [46] [A.G. Bailey, C.P. Lowe, MILCH SHAKE: an ef](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref46)ficient method for constraint dy[namics applied to alkanes, J. Comput. Chem. 30 \(2009\) 2485](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref46)e[2493.](http://refhub.elsevier.com/S0223-5234(16)30937-0/sref46)

c1ccccc1

c1ccccc1